lib/cartopy/mpl/feature_artist.py

```diff
@@ -113,21 +110,31 @@ def __init__(self, feature, **kwargs):
             color = self._kwargs.pop('color')
             self._kwargs['facecolor'] = self._kwargs['edgecolor'] = color
 
+        self.set_paths([])
```

Comment:
`set_paths` was added at mpl 3.8 so this will need more work for older versions.